pkg/controller/config/config_controller.go

```diff
@@ -722,3 +731,20 @@ func sourceBasedOnRecursion(path string) mf.Source {
 	}
 	return mf.Path(path)
 }
+
+func createAddonsPipelines(mgr manager.Manager) (mf.Manifest, error) {
+	addonsPath := filepath.Join("pkg/utils/addons", "pipelinetemplate.yaml")
```

Comment:
shal we move this template also in to `deploy/resources/addons` as this is something which might change in each release.

Reply:
I think putting `pipelinetemplate.yaml` in   `deploy/resources/addons` will cause issues because this template is an incomplete yaml and when operator try to apply this yaml will give an error

Reply:
i see, 
then could you place it in `deploy/resources/04-templates`

Reply:
Yes, this can be done.